cluster/saltbase/salt/kube-addons/init.sls

```diff
@@ -81,6 +81,42 @@ addon-dir-create:
     - makedirs: True
 {% endif %}
 
+{% if pillar.get('enable_cluster_registry', '').lower() == 'true' %}
```

Comment:
Heh, you actually succeeded where many have failed. GKE will happily just keep trucking with this change (and assume it's off). We initially won't pass the kube-env correctly, so configure-vm will whine, but that particular block in configure-vm doesn't actually errexit from nounset (which is an accidental "feature"), so `enable_cluster_registry` just ends up empty.